HED-schema.mediawiki

```diff
@@ -137,16 +148,12 @@ From version 2.2, HED adheres to http://semver.org/ versioning.
 * IAPS <nowiki>[International Affective Picture System]  </nowiki> 
 * IADS <nowiki> [International Affective Digital Sounds]</nowiki> 
 * SAM <nowiki> [The Self-Assessment Manikin]</nowiki> 
+** Music
```

Comment:
Is this supposed to be under SAM?

Reply:
No SAM is a set of images (like the pain smiley faces) for assessing mode.  However, I am going to move all under multimedia in this round